tools/make-source

```diff
@@ -42,6 +42,6 @@ else
     builddir=.
 fi
 
-make -C $builddir --silent dist-gzip distdir=cockpit-wip 1>&2
+make -j8 -C $builddir --silent dist-gzip distdir=cockpit-wip 1>&2
```

Comment:
And also, it really is just a memory issue. nodejs is notoriously single threaded ... even on one core, hyperthreading, and context switching in parallel would have minor speed up wins.

The real issue for reducing 8 to say 4 ... is memory. I'd like to try it at 8 and see if we have problems.